Dockerfile

```diff
@@ -10,15 +10,13 @@ RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
 RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
 RUN apt-get update
 
-WORKDIR /opt
-COPY google-cloud-sdk /opt/google-cloud-sdk
-
-RUN /opt/google-cloud-sdk/bin/gcloud auth list
-
 # Copy the kernel into the container
 WORKDIR /swift-apis
 COPY . .
 
+RUN if test -f google-cloud-sdk; then mv google-cloud-sdk /opt/google-cloud-sdk; fi
+RUN if test -f /opt/google-cloud-sdk/bin/gcloud; then /opt/google-cloud-sdk/bin/gcloud auth list; fi
```

Comment:
Can you make this a single `RUN` please, this is creating an unnecessary intermittent layer that is unnecessary.